tachyon/math/elliptic_curves/msm/algorithms/icicle/icicle_msm.h

```diff
@@ -5,13 +5,15 @@
 
 #include "third_party/icicle/include/fields/id.h"
 
+#include "tachyon/device/gpu/gpu_memory.h"
+#include "tachyon/math/elliptic_curves/affine_point.h"
```

Comment:
Is `jacobian_point.h` unneeded?

Reply:
This file has been refactored to use only `ProjectivePoint`, `jacobian_point.h` is not needed and `affine_point.h` also should be removed.